packages/ag-charts-community/src/api/preset/sparkline.ts

```diff
@@ -245,6 +275,9 @@ function gridLinePreset(opts: AgSparklineAxisOptions | undefined, defaultEnabled
         gridLineOpts.width = opts?.strokeWidth;
         gridLineOpts.enabled ??= true;
     }
+    if (sparkOpts.type === 'bar' && sparkOpts.direction !== 'horizontal') {
+        gridLineOpts.enabled ??= true;
+    }
```

Comment:
Could we just pass in `direction` here since we're working it out below?

Reply:
I don't think the change is trivial, especially when direction really only applies to bar sparklines here.